src/Perscom/Entity/AwardNomination.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Forumify\PerscomPlugin\Perscom\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+use Forumify\Core\Entity\IdentifiableEntityTrait;
+use Forumify\PerscomPlugin\Perscom\Repository\AwardNominationRepository;
+
+#[ORM\Entity(repositoryClass: AwardNominationRepository::class)]
+#[ORM\Table('award_nomination')]
```

Comment:
Non-platform tables should start with a prefix to prevent collisions, for this plugin, that is `perscom_`.